src/activity_handlers.cpp

```diff
@@ -2284,6 +2284,10 @@ void activity_handlers::oxytorch_finish( player_activity *act, player *p )
         here.ter_set( pos, t_mdoor_frame );
         here.spawn_item( pos, itype_steel_plate, rng( 0, 1 ) );
         here.spawn_item( pos, itype_steel_chunk, rng( 3, 8 ) );
+    } else if( ter == t_wall_metal ) {
+        here.ter_set( pos, t_dirt );
```

Comment:
it's `t_null` (not yet sure what that means) in https://github.com/CleverRaven/Cataclysm-DDA/blob/3d138017b8381d1dc640321d65ac015506e39d85/data/json/furniture_and_terrain/terrain-walls.json#L438

Reply:
Is it `t_null`? There is no `t_null` ( there is `f_null` ) in the rest of the code around here in the acetylene section. IDK what `t_null` does to, so I just looked around the code and saw that they used `t_dirt`.
I mean I can change it if you want and think that it will work well.

Reply:
Not sure but considering soil should be hard to encounter underground I'd change it to metal floor.

Reply:
What if it isn't underground?

Reply:
I imagine the cutting as not removing the whole wall, but just making a hole enough to pass through. As such, the "floor" would still be metal.

Reply:
>  I'd change it to metal floor.

I would be fine with this but I think there is now way to remove a metal floor ( unless you use a sledgehammer maybe but IDK if you can remove it with that tool, or mine under the floor and dig up )? 
As such if a player makes a mistake and builds a metal wall and then when it gets removed he won't be able to dig the shallow pit that is necessary for building walls, or he will get stuck with a tile that is very hard to remove if he wants to build something there ( unless a sledgehammer can be used ).

I agree that dirt is out of place most of the time and if you want I will change it to a metal floor, just making sure that we are on the same page about what this will do when constructing bases or other situations.

Also here is a response about t_null:
![image](https://user-images.githubusercontent.com/51816321/103142214-291cdd00-4708-11eb-8e32-ee88fbd1cd02.png)

Reply:
What do you think about adding a metal floor deconstruction? What is the terrain set to currently (under what conditions) if you bash it hard enough, did you read the `map::bash_ter_furn`?

Reply:
`t_scrap_wall_halfway`, which is supposed to be (scrap) metal, bashes to `t_pit_shallow`. (Would it work to convert low-grade metal walls into `t_scrap_wall_halfway` after the use of an acetylene torch?)

Reply:
I have changed the terrain type in question to be `t_scrap_wall_halfway` instead of `t_dirt`.